resources/shaders/dxr_global.hlsl

```diff
@@ -24,7 +24,7 @@
 #define SOFT_SHADOWS
 #define MAX_SHADOW_SAMPLES 1
 
-//#define PERFECT_MIRROR_REFLECTIONS
+#define PERFECT_MIRROR_REFLECTIONS
```

Comment:
Doing this should disable rough reflections. PERFECT_MIRROR_REFLECTIONS shouldn't normally be defined.

Reply:
it does not.